parlai/utils/torch.py

```diff
@@ -65,11 +65,9 @@ def atomic_save(state_dict: Any, path: str) -> None:
 def padded_tensor(
     items: List[Union[List[int], torch.LongTensor]],
     pad_idx: int = 0,
-    use_cuda: bool = False,
```

Comment:
this will almost certainly break some internal stuff but i suppose those can be dealt with individually